src/core_plugins/kibana/public/management/sections/indices/_indexed_fields.js

```diff
@@ -25,6 +27,7 @@ uiModules.get('apps/management')
         { title: 'format' },
         { title: 'analyzed', info: 'Analyzed fields may require extra memory to visualize' },
         { title: 'indexed', info: 'Fields that are not indexed are unavailable for search' },
+        { title: 'exclude', info: 'Fields that are excluded from _source when it is fetched' },
```

Comment:
The title should probably be `excluded` instead of `exclude` to match `indexed` and `analyzed`.